internal/charmstore/archive.go

```diff
@@ -14,7 +14,10 @@ import (
 	"gopkg.in/juju/charmstore.v5-unstable/internal/blobstore"
 )
 
-type archiverTo interface {
+// ArchiverTo can be used to archive a charm or bundle's
```

Comment:
Because it's mentioned in the docs for AddCharmWithArchive and AddBundleWithArchive
and it seems mean to mention a type that's not visible in the godocs.